src/handler.py

```diff
@@ -154,8 +164,35 @@ def control_db_utilization(event, context):
         However, we know there is a backlog to work through so we need to force the db to maximum size
         by issuing a fake high-cpu alarm.
         """
-        logger.info(f"os.environ {os.environ}")
-        execute_recover_machine({}, {})
+        logger.info(f"The circuit breaker has calmed down.  Let's try to ramp things up.")
```

Comment:
I'm a bit confused. Isn't this the circuit breaker?

Reply:
I meant the error handler, will change.